devscripts/run_tests.py

```diff
@@ -40,8 +39,10 @@ def run_tests(*tests, pattern=None, ci=False):
     elif run_download:
         arguments.extend(['-m', 'download'])
     else:
-        arguments.extend(
-            f'test/test_download.py::TestDownload::test_{test}' for test in tests)
+        for test in tests:
+            if '/' not in test and '::' not in test:
```

Comment:
woudn't checking for `/` be enough? do we want to allow passing a full test module?
```suggestion
            if '/' not in test:
```

Reply:
Yes I think you are right, everything will have to use the full qualified name, I'll change it